src/main/java/com/revature/beans/User.java

```diff
@@ -199,6 +212,74 @@ public User(int userId, @NotBlank String userName, Batch batch, @NotBlank String
 		this.wZip = wZip;
 		this.wState = wState;
 	}
+	
+	public User(@Valid @NotBlank @Size(min = 3, max = 12) @Pattern(regexp = "^\\w+\\.?\\w+$") String userName,
+			Batch batch,
+			@Valid @NotBlank @Size(max = 30) @Pattern(regexp = "^[a-zA-Z\\u00C0-\\u017F]+[- ]?[a-zA-Z\\u00C0-\\u017F]+$") String firstName,
+			@Valid @NotBlank @Size(max = 30) @Pattern(regexp = "^[a-zA-Z\\u00C0-\\u017F]+[- ]?[a-zA-Z\\u00C0-\\u017F]+$") String lastName,
+			@NotBlank @Email @Pattern(regexp = "^\\w+\\.?\\w+@\\w+\\.[a-zA-Z]{2,4}$") String email,
+			@NotBlank @Pattern(regexp = "^\\(?([0-9]{3})\\)?[-.\\s]?([0-9]{3})[-.\\s]?([0-9]{4})$") String phoneNumber,
+			boolean isDriver, boolean isActive, boolean isAcceptingRides, @NotBlank String hAddress,
+			@NotBlank String hCity, @NotBlank String hZip, @NotBlank String hState, @NotBlank String wAddress,
+			@NotBlank String wCity, @NotBlank String wZip, @NotBlank String wState, double distance, double duration,
+			Car car) {
```

Comment:
There's a lot going on here, perhaps the builder pattern can be used instead? Also is the full work address necessary here? It seems like it will just be a selection of a few locations

Reply:
I suspect this constructor is useless so I commented it out and ran all features and they all worked. So I removed this constructor.